### PR TITLE
feat: getinvoices for lnbits connector

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

Adds invoice fetching support for LNBits

**Note:** We don't pass in `custom_records` as of now, please lmk if there's a way to extract from LNBits invoice response

### Link this PR to an issue

#46

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes (If any)

Screenshot from 2022-09-22 15-00-37

### How has this been tested?

Manually using LNBits account

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/extension/background-script/connectors/lnbits.ts [hunk 1]
```diff
@@ -64,13 +66,69 @@ class LnBits implements Connector {
...
-      `Not yet supported with the currently used account: ${this.constructor.name}`
-    );
-    return new Error(
-      `${this.constructor.name}: "getInvoices" is not yet supported. Contact us if you need it.`
+  /*
+  LNBits Swagger Docs: https://legend.lnbits.org/devs/swagger.html#/default/api_payments_api_v1_payments_get
+  Sample Response:
+    [
+      {
+        "checking_id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
```
Comment (escapedcat) [inline src/extension/background-script/connectors/lnbits.ts:74]: @bumi you know if `custom_records` are hidden in here somewhere?

Comment (bumi) [reply src/extension/background-script/connectors/lnbits.ts:74]: in the checking id? 
no, I guess that's their internal id.

where are the docs for this call?

Comment (im-adithya) [reply src/extension/background-script/connectors/lnbits.ts:74]: https://legend.lnbits.org/devs/swagger.html#/default/api_payments_api_v1_payments_get

Comment (bumi) [reply src/extension/background-script/connectors/lnbits.ts:74]: it does not describe the response data, does it?

Comment (escapedcat) [reply src/extension/background-script/connectors/lnbits.ts:74]: Maybe you need to authorize first?:


On "try out":


This shows some info:
https://legend.lnbits.org/devs/swagger.html#/default/api_payments_create_api_v1_payments_post

Comment (im-adithya) [reply src/extension/background-script/connectors/lnbits.ts:74]: Here's the actual response:
Screenshot from 2022-09-30 18-24-26

Comment (escapedcat) [reply src/extension/background-script/connectors/lnbits.ts:74]: @bumi looks liek no `custom_records` available?  
Should we merge this in first to provide general support first?

**Comment by bumi** [comment]
utACK

**Pull request merged**